src/components/external/DisputeManagement/components/DefendDisputeFlow/DefendDisputeReason.tsx

```diff
@@ -132,6 +134,14 @@ export const DefendDisputeReason = () => {
         [defenseReasonConfig, i18n, selected]
     );
 
+    const defendDisputeLabel = useMemo(
+        () =>
+            isRequestForInformation
+                ? i18n.get('disputes.management.defend.rfi.selectDefenseReason')
```

Comment:
Changing `rfi` to `requestForInformation` for consistency.

```suggestion
                ? i18n.get('disputes.management.defend.requestForInformation.selectDefenseReason')
```